x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -135,12 +123,244 @@ with its version bumped to 2.
 Afterwords, any indices set to `my_policy` will execute against version 2 of
 the policy.
 
-=== B. Update policy with existing index in the warm phase
+B. Effects of updating a specific phase's definition with an index currently executing the policy.
```

Comment:
I actually think it would be good to have these as sections and have a paragraph introducing the method instead of bullet points. I also think we can lose the `A`, `B`, `C` if these are sections

Reply:
agreed. switched